linux/deb_run.sh

```diff
@@ -0,0 +1,38 @@
+# install Python3.8.2
+#https://linuxize.com/post/how-to-install-python-3-8-on-debian-10/
+sudo apt-get update -y && \
+sudo apt-get install -y build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libsqlite3-dev libreadline-dev libffi-dev curl liblzma-dev libbz2-dev && \
+curl -O https://www.python.org/ftp/python/3.8.2/Python-3.8.2.tar.xz && \
+tar -xf Python-3.8.2.tar.xz && \
+cd Python-3.8.2 && \
+./configure --enable-optimizations && \
+make -j `nproc` && \
+sudo make altinstall && \
+python3.8 --version
```

Comment:
why not just call deb_install_python3.8.sh?

Reply:
I was going to do that but then I was thinking that maybe this script would be ran outside the project so both scripts would need to be available side by side.
anyway I didn't test it yet as I'm working on making the Docker image work first